webapp/titanembeds/templates/embed.html.j2

```diff
@@ -374,6 +374,13 @@
 
     <script type="text/javascript" src="{{ url_for('static', filename='js/embed.js') }}"></script>
     
+    <!--Specify icons for Apple devices (can also be used as fallback for Android)-->
+    <link rel="apple-touch-icon" href="https://titanembeds.com/static/img/titanembeds_square.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="https://titanembeds.com/static/img/appicons/icon_60x60@3x.png" />
```

Comment:
For the URLs, could you make it a relative urls?
Remember that other people can run the Titan project and they have different domains. Just drop the https://titanembeds.com from the URl and it would be great!